.vscode/settings.json

```diff
@@ -7,6 +7,6 @@
             "source.organizeImports": true,
             "source.sortMembers": true
         },
-        "editor.formatOnSave": true
+        "editor.formatOnSave": false
```

Comment:
Why are you changing this configuration. Should be useful to have it enabled.

Reply:
I find it annoying, given that my IDE config does not pick up prettier settings automatically. I will revert this config, it slipped in with other changes